projects.go

```diff
@@ -34,43 +34,45 @@ type ProjectsService struct {
-	SnippetsEnabled               bool                 `json:"snippets_enabled"`
-	ContainerRegistryEnabled      bool                 `json:"container_registry_enabled"`
-	CreatedAt                     *time.Time           `json:"created_at,omitempty"`
-	LastActivityAt                *time.Time           `json:"last_activity_at,omitempty"`
-	CreatorID                     int                  `json:"creator_id"`
-	Namespace                     *ProjectNamespace    `json:"namespace"`
-	Permissions                   *Permissions         `json:"permissions"`
-	Archived                      bool                 `json:"archived"`
-	AvatarURL                     string               `json:"avatar_url"`
-	SharedRunnersEnabled          bool                 `json:"shared_runners_enabled"`
-	ForksCount                    int                  `json:"forks_count"`
-	StarCount                     int                  `json:"star_count"`
-	RunnersToken                  string               `json:"runners_token"`
-	PublicBuilds                  bool                 `json:"public_builds"`
-	OnlyAllowMergeIfBuildSucceeds bool                 `json:"only_allow_merge_if_build_succeeds"`
-	LFSEnabled                    bool                 `json:"lfs_enabled"`
-	RequestAccessEnabled          bool                 `json:"request_access_enabled"`
-	SharedWithGroups              []struct {
+	ID                                        int                  `json:"id"`
+	Description                               string               `json:"description"`
+	DefaultBranch                             string               `json:"default_branch"`
+	Public                                    bool                 `json:"public"`
+	VisibilityLevel                           VisibilityLevelValue `json:"visibility_level"`
+	SSHURLToRepo                              string               `json:"ssh_url_to_repo"`
+	HTTPURLToRepo                             string               `json:"http_url_to_repo"`
+	WebURL                                    string               `json:"web_url"`
+	TagList                                   []string             `json:"tag_list"`
+	Owner                                     *User                `json:"owner"`
+	Name                                      string               `json:"name"`
+	NameWithNamespace                         string               `json:"name_with_namespace"`
+	Path                                      string               `json:"path"`
+	PathWithNamespace                         string               `json:"path_with_namespace"`
+	IssuesEnabled                             bool                 `json:"issues_enabled"`
+	OpenIssuesCount                           int                  `json:"open_issues_count"`
+	MergeRequestsEnabled                      bool                 `json:"merge_requests_enabled"`
+	ApprovalsBeforeMerge                      int                  `json:"approvals_before_merge"`
```

Comment:
Yeah, it's actually not documented on the Projects API. It's description is on the [Merge Requests API](https://docs.gitlab.com/ee/api/merge_requests.html#create-mr) and there's a mention there of its presence on the project.